virt/release_notes/virt-4-16-release-notes.adoc

```diff
@@ -111,7 +114,7 @@ On {op-system-base-full} 9, xref:../../virt/monitoring/virt-exposing-downward-me
 * VMs require a minimum of 1 GiB of allocated memory to enable memory hotplug. If a VM has less than 1 GiB of allocated memory, then memory hotplug is disabled.
 
 //CNV-34681: runbooks moved to openshift/runbooks repo
-* Runbooks for {VirtProductName} alerts are now maintained only in the https://github.com/openshift/runbooks/tree/master/alerts/openshift-virtualization-operator[`openshift/runbooks` git repository] and have been removed from the documentation. 
+* Runbooks for {VirtProductName} alerts are now maintained only in the https://github.com/openshift/runbooks/tree/master/alerts/openshift-virtualization-operator[`openshift/runbooks` git repository] and have been removed from the documentation.
```

Comment:
```suggestion
* Runbooks for {VirtProductName} alerts are now only maintained in the https://github.com/openshift/runbooks/tree/master/alerts/openshift-virtualization-operator[`openshift/runbooks` git repository] and have been removed from the documentation.
```

![Screenshot from 2024-07-03 16-29-12](https://github.com/openshift/openshift-docs/assets/57954076/d409c804-ea51-46dc-a48b-a60661d1e7c5)

Reply:
I did not write or update the note about runbooks, so I will leave as is.

Reply:
But I believe that the placement of 'only' in this sentence is correct--the alerts are maintained only in the location specified.

Reply:
Thanks for clarifying.